test/Distributed/Runtime/distributed_no_transport_boom.swift

```diff
@@ -36,7 +36,7 @@ struct FakeTransport: ActorTransport {
     fatalError("not implemented:\(#function)")
   }
 
-  func resolve<Act>(_ identity: Act.ID, as actorType: Act.Type)
+  func resolve<Act>(_ identity: AnyActorIdentity, as actorType: Act.Type)
```

Comment:
just compile fixes of tests that we want to re-enable (long story why they were not)